srml/executive/src/mock.rs

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Test utilities
+
+#![cfg(test)]
+
+use super::*;
+use runtime_io;
+use substrate_primitives::{Blake2Hasher};
+use srml_support::{impl_outer_origin, impl_outer_event, impl_outer_dispatch, parameter_types};
+use primitives::traits::{IdentityLookup, BlakeTwo256};
+use primitives::testing::{Header, Block};
+use system;
+pub use balances::Call as balancesCall;
+pub use system::Call as systemCall;
+use node_runtime::impls::FeeMultiplierUpdateHandler;
```

Comment:
Maybe move `FeeMultiplierUpdateHandler` to core?

Reply:
My vision was that eventually, we would place all of the structs that implement some `Convert` (and other traits commonly used in runtime as associated types) and are passed to traits as associated `type` would be placed next to the runtime definition in `impl.rs`. Since these are user configurable (or at least intended to be), I'd say they have to be somewhere in node/runtime.